test/integration/emitter_test.cpp

```diff
@@ -266,16 +266,23 @@ TEST_F(EmitterTest, SimpleLongKey) {
 }
 
 TEST_F(EmitterTest, SingleLongKey) {
+  const std::string& shortKey = std::string(1024, 'a');
```

Comment:
1) it’s more idiomatic to have this be a value, not a reference
2) you can construct it directly (rather than use =)

Reply:
ok, I have modified it.